Marlin/src/HAL/HAL_STM32F1/HAL_spi_STM32F1.cpp

```diff
@@ -104,7 +104,7 @@ void spiInit(uint8_t spiRate) {
  * @details
  */
 uint8_t spiRec(void) {
-  uint8_t returnByte = SPI.transfer(0xFF);
+  uint8_t returnByte = SPI.transfer(ff);
```

Comment:
What does `ff` refer to?

Reply:
its a static const in framework, = 0xff but when unused makes a warning

Reply:
I see. So, it's of the exact type that `SPI.transfer` is expecting.